app/scripts/controllers/edit/buildConfig.js

```diff
@@ -335,42 +333,50 @@ angular.module('openshiftConsole')
         var builderRef = $filter('imageObjectRef')(buildConfigFrom, $scope.projectName);
         return imageChangeRef === builderRef;
       }
+      // Applies to webhook triggers
+      function constructWebhookTrigger(trigger, isDisabled) {
+        return {disabled: isDisabled, data: trigger};
+      }
+      // Applies to image and config change triggers
+      function constructChangeTrigger(trigger, isPresent) {
+        return {present: isPresent, data: trigger};
+      }
```

Comment:
Sorry, by "inline" I meant not have the functions, just create the objects below.